src/glad.c

```diff
@@ -771,7 +771,12 @@ PFNWGLGETPROCADDRESSPROC_PRIVATE gladGetProcAddressPtr;
 
 static
 int open_gl(void) {
+#if defined(RTC_WINDOWS_UNIVERSAL)
```

Comment:
Would this be better to use a compiler defined ifdef so it could be pushed upstream?

Reply:
@dotMorten do you happen to know of any?

Reply:
This change is code by glad, so it won't be possible to push upstream. I've made an issue for this in the upstream repo: https://github.com/Dav1dde/glad/issues/141